10_nwis_pull.yml

```diff
@@ -110,14 +115,18 @@ targets:
 
   10_nwis_uv_pull_tasks.yml:
     command: create_nwis_pull_makefile(makefile=target_name, task_plan=nwis_uv_pull_plan,
-      final_targets = I('10_nwis_pull/out/nwis_uv_data.rds.ind'))
+      final_targets = I('10_nwis_pull/tmp/nwis_uv_data.rds.ind'))
 
-  10_nwis_pull/out/nwis_uv_data.rds.ind:
+  10_nwis_pull/tmp/nwis_uv_data.rds.ind:
     command: loop_tasks(task_plan=nwis_uv_pull_plan, task_makefile='10_nwis_uv_pull_tasks.yml',
       num_tries=I(30), sleep_on_error=I(20))
     depends:
       - 10_nwis_pull/src/nwis_pull.R
```

Comment:
This pattern is a bit outdated. We have started combining the "create plan" + "create makefile" + "loop through tasks" steps into a single function/target, often named `do_[something]_tasks`. It just streamlines the main makefiles and also cleans up the task makefiles that aren't checked in to GitHub anyways. 

[Here is an example of a function](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/1_fetch/src/do_gw_fetch.R) that plans, creates, and executes a task makefile. And [here is the corresponding target](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/1_fetch.yml#L39-L46) in the main makefile.

This is just as an FYI. I don't think it isn't necessary to implement since this is already building.